src/rendering/text/TextView.ts

```diff
@@ -59,17 +60,25 @@ export class TextView implements View
     private _bounds: [number, number, number, number] = [0, 1, 0, 0];
     private _boundsDirty = true;
     private _text: string;
+    private readonly _renderMode: string;
 
     constructor(options: TextViewOptions)
     {
         this.text = options.text ?? '';
-        this._style = options.style instanceof TextStyle ? options.style : new TextStyle(options.style);
 
-        const renderMode = options.renderMode ?? this._detectRenderType(this._style);
+        const renderMode = options.renderMode ?? this._detectRenderType(options.style);
```

Comment:
`this._detectRenderType` needs to be updated to include `html` text. If you provide a `HTMLTextStyle` then we can auto detect it